src/OrchardCore.Modules/Orchard.Tokens.Content/Services/ISlugService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Tokens.Content.Services
+{
+    public interface ISlugService
+    {
+        /// <summary>
+        /// Transforms specified text to the form suitable for URL slugs
+        /// </summary>
+        /// <param name="text">the text to transform</param>
```

Comment:
Uppercase at the beginnings, dot at the ends. Welcome to the "nit picking" level of reviews, that hurts at first, but after that you'll either not add comments or do it perfectly ;)